src/main/java/figure/Triangle.java

```diff
@@ -0,0 +1,69 @@
+package figure;
+
+public class Triangle extends AbstractFigure {
+    private double catet1;
+    private double catet2;
+    private String color;
+
+    public Triangle(double catet1, double catet2, String color) {
+        this.catet1 = catet1;
+        this.catet2 = catet2;
```

Comment:
add hypotenuse  as well

Reply:
added hypotenuse and constructor with it